makefile

```diff
@@ -33,7 +33,7 @@ all: $(TARGET)
 ios: libbinn.a libbinn.dylib
 
 libbinn.so.1.0: binn.o
-	$(CC) -shared -Wl,-soname,$(LINK1) -o $@ $^
+	$(CC) -shared -Wl,-soname,$(LINK1),-rpath,$(PREFIX)/lib/ -o $@ $^
```

Comment:
why to add an rpath to the library? the applications that use the library are those who would need an rpath to find the library

Reply:
thats my mistake..